priv/data/affiliate/affiliates.exs

```diff
@@ -0,0 +1,27 @@
+[
+  %{
+    name: "Elixir",
+    link: "https://elixir-lang.org",
+    icon: "/images/affiliate/elixir-icon.png",
```

Comment:
These paths do not exist, you could simply change the lines to point to existing icons in the case of erlang and elixir (those live under `community/languages` and as SVG. For ow2 you could extract the svg from `lib/erlef_web/templates/layout/footer.html.eex` and put it in a file then reference it. `assets/static/images/community/affiliates` makes sense to me.